cmd/gpu_plugin/README.md

```diff
@@ -50,6 +55,74 @@ The following sections detail how to obtain, build, deploy and test the GPU devi
 
 Examples are provided showing how to deploy the plugin either using a DaemonSet or by hand on a per-node basis.
 
+### Prerequisites
+
+Access to a GPU device requires firmware, kernel and user-space
+drivers supporting it.  Firmware and kernel driver need to be on the
+host, user-space drivers in the GPU workload containers.
+
+GPU devices supported by the current kernel can be listed with:
+```
+head /sys/class/drm/card[0-9]/device/uevent
+```
+
+Ones with `DRIVER=i915` are Intel ones supported by this plugin. GPUs
+with display support belong to `PCI_CLASS=30000`, and ones without
+display to `PCI_CLASS=38000`:
+```
+==> /sys/class/drm/card0/device/uevent <==
+DRIVER=i915
+PCI_CLASS=38000
+PCI_ID=8086:56C0
+PCI_SUBSYS_ID=8086:4905
+PCI_SLOT_NAME=0000:4d:00.0
+MODALIAS=pci:v00008086d000056C0sv00008086sd00004905bc03sc80i00
+```
```

Comment:
I'd decrease the level of details to just a one-liner like so:
```
grep "DRIVER=i915" /sys/class/drm/card?/device/uevent
/sys/class/drm/card0/device/uevent:DRIVER=i915
/sys/class/drm/card1/device/uevent:DRIVER=i915
```
And leave out the details about the display and non-display cards. They could be noted somewhere below but I don't think it's important to mention in "Prerequisites". Maybe later in some debugging notes?

Reply:
When display vs. non-display GPU labeling gets fixed for all GPU plugin install methods, omitting those details here is fine.

I probably need to discuss finding the correct PCI ID to force probe in the kernel section, as it won't show up here until i915 actually probes the device.  I guess this is the best way: `lspci | grep -e VGA -e Display | grep Intel`?

Reply:
I suppose the lspci-grep combination is the best/quickest way to detect if the GPU PCIe devices are available in the system. No point in adding drivers if that's not the case.

Reply:
> started to wonder whether `uevent` and `lspci` examples should be of the same set of GPUs not to confuse reader?
I think it would make sense.

Reply:
I'll update that at the same time with the KMD section update (after pre-built binary packages for KMD are available, and WIP: prefix can be removed from this PR).